### PR TITLE
Type tag construction 

### PR DESCRIPTION
**Pull request by georgemitenkov**
## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: aptos-move/aptos-gas-schedule/src/ver.rs [hunk 1]
```diff
@@ -76,4 +78,5 @@ pub mod gas_feature_versions {
     pub const RELEASE_V1_13: u64 = 18;
     pub const RELEASE_V1_14: u64 = 19;
     pub const RELEASE_V1_15: u64 = 20;
+    pub const RELEASE_V1_16: u64 = 21;
```
Comment (sherry-x) [inline aptos-move/aptos-gas-schedule/src/ver.rs:81]: could you please add a couple more numbers while you at it? thank you!

**Review by sherry-x — APPROVED** [review]

File: aptos-move/aptos-gas-schedule/src/ver.rs (hunk 1 above)
Comment (georgemitenkov) [reply aptos-move/aptos-gas-schedule/src/ver.rs:81]: For previous releases?

Comment (sherry-x) [reply aptos-move/aptos-gas-schedule/src/ver.rs:81]: for future releases :D

Comment (georgemitenkov) [reply aptos-move/aptos-gas-schedule/src/ver.rs:81]: done

**Review by vgao1996 — APPROVED** [review]

**Pull request merged**